jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -626,7 +626,9 @@ def export_extract(self, **kwargs):
             self.update_marks(step='ext')
 
         trace = self._get_ext_trace()
-        inp_sp2d = self._get_ext_input_spectrum().flux.value
+
+        # FIXME: Passing in Quantity turns MJy into MJy2. But not passing it in gives DN.
+        inp_sp2d = self._get_ext_input_spectrum().flux
```

Comment:
@kecnry , here is the unit stuff I mentioned in Slack.

Reply:
I should probably back this out and leave it at DN?

Reply:
@kecnry , I cannot find the ticket that mentions this separately but I remember you referred to it at planning. If I misheard, then I will create a new ticket. Thanks!

Reply:
to summarize - I think there are two issues here, both of which are present in main as well and for datasets outside the scope of this PR, so I agree that these changes can be stripped from this PR and handled separately:
* glue-astronomy is dropping units and so they're not available at this point in the code.  https://github.com/glue-viz/glue-astronomy/pull/78 will _hopefully_ fix this
* specreduce does not currently take specutils objects as input.  That will hopefully be supported soon, in which case the units should be maintained through the specreduce calls.  Alternatively, if the point above gets in first, we could always force the units to be re-attached (within the plugin logic) after the specreduce calls.

Reply:
OK, I left a comment at https://github.com/astropy/specreduce/issues/130#issuecomment-1244330983 . I will back this change out.